cloud/cmd/csidriver/app/options/options.go

```diff
@@ -41,6 +41,5 @@ func (o *CSIDriverOptions) Flags() (fss cliflag.NamedFlagSets) {
 	fs.StringVar(&o.DriverName, "drivername", "csidriver", "name of the driver")
 	fs.StringVar(&o.NodeID, "nodeid", "", "node id determines which node will be used to create/delete volumes")
 	fs.StringVar(&o.KubeEdgeEndpoint, "kubeedge-endpoint", "unix:///kubeedge/kubeedge.sock", "kubeedge endpoint")
-	fs.StringVar(&o.Version, "version", "dev", "version")
```

Comment:
Do we still need line 29 `Version`?

Reply:
yes, we do. It is being used in [csidriver package](https://github.com/kubeedge/kubeedge/blob/e60886d889b26a4d9e8886991475bdf6ba0423f6/cloud/pkg/csidriver/server.go#L59)